### ISSUE TITLE
[Feature Request][TF2] Disable all developer console warnings

### ISSUE THREAD

**Issue by Sreap** [ValveSoftware/Source-1-Games#4536]
I'm requesting a convar to disable all of the red warning text entirely in the developer console. I'm aware that con_filter_text_out exists but it only seems to support one string of characters and not viable if a map is spamming different warnings. I also wouldn't want to filter out non-warnings by mistake.

Example of console warning spam from a map that isn't mine and won't ever apply to me:
![image](https://user-images.githubusercontent.com/63338919/204083287-2cedb7cf-a0af-4cea-8352-0d8ebb522e53.png)

**Comment by RicochetYT** [comment]
Yeah, I've thought about this. It would be nice if there was a cvar that could disable error messages OR filter multiple strings out.
My only complaint with using con_filter_text_out is that it does a strcmp every time a console message is sent to my understanding, so it would be nicer to have something else.
